pybliometrics/scopus/author_retrieval.py

```diff
@@ -37,6 +37,11 @@ def affiliation_history(self):
         path = ["author-profile", "affiliation-history", "affiliation"]
         return parse_affiliation(chained_get(self._json, path))
 
+    @property
+    def alias(self):
+        """Possible alternative Scopus Author Profile IDs, if any."""
```

Comment:
I think it's better to not call them "alternative", since they're in fact possible new ones. What about this text? """Set of possible new Scopus Author Profile IDs in case the profile has been merged."""